DOCKER.md

```diff
@@ -1,45 +1,47 @@
 # Dockerized instance of EpochTalk
 
-## Quickstart example
+## Docker Compose
 
-1. Create the Docker EpockTalk instance:
+The `docker-compose` deployment contains resources for:  Postgres, Redis, and
+[Epoch](https://github.com/epochtalk/epoch).  The recipe for installation is
+contained in the [docker-compose.yml](./docker-compose.yml) file
 
-  `docker build -t epochtalk . && docker run -d -P --name epochtalk epochtalk`
+### Configuration
 
-2. Wait until the build is finished by checking:
+Create `docker.env` with whatever options you would like.
 
-  `docker logs epochtalk`
+By default, `docker-compose` will expose port `8080` on the docker network.
 
-3. Access the server by finding the right ip/port:
 
-  `echo $(docker-machine ip default):$(docker port epochtalk 8080 | cut -f 2 -d :)`
+### Bring up the EpochTalk service
 
-## Dockumentation
+`docker-compose up`
 
-* start up the thing
+On the first run, this command will download and install containers for
+dependencies, and build the EpochTalk image. Because of this, it will take a bit
+longer to start.  Subsequent startups should be quicker.
```

Comment:
Broh, be consistent, you have two spaces between sentences here and only one on the line before